Develop

AustinZhu opened the pull request:
add full-stack config

OscarWang114 commented:
My latest commit on the develop branch wasn't added. Possibly due to the shutdown earlier. I'll close this PR and open a new one.

AustinZhu committed:
feat: add webapp stack

AustinZhu committed:
fix: disable test

AustinZhu committed:
feat: add basic api framework

AustinZhu committed:
feat: add api
fix: amplify domain
feat: add lambda

AustinZhu committed:
feat: add syllabus data pipeline

AustinZhu committed:
format: reformat code
fix: modify webapp

AustinZhu committed:
feat: add api lambda integration
feat: add event rules
refactor: improve code structure

AustinZhu committed:
refactor: more decoupling and code reuse

AustinZhu committed:
refactor: lambda-functions.ts, task-managers.ts, data-pipeline.ts

AustinZhu committed:
fix: fix bugs
feat: add operation.ts auth-endpoint.ts service-monitor.ts
feat: cdk app validation by travis
docs: README.md

AustinZhu committed:
doc: add picture

AustinZhu committed:
refactor: decoupling and more abstraction

AustinZhu committed:
refactor: decoupling and more abstraction
feat: add table
feat: add travis scripts

AustinZhu committed:
add functions and dummy files

AustinZhu committed:
feat: add more table, api and pipelines

AustinZhu committed:
fix: attribute error

AustinZhu committed:
feat: add travis deploy

AustinZhu committed:
fix: deploy parameter
